.github/workflows/ci.yml

```diff
@@ -789,8 +789,8 @@ jobs:
 
           universal_zip_file_name=$(basename "${UNIVERSAL_ZIP}")
           if [ -n "$universal_zip_file_name" ]; then
-            echo universal_zip_url_latest="$gcs_artifacts_path_latest/$universal_zip_file_name" >> "$GITHUB_OUTPUT"
-            echo universal_zip_url_commit="$gcs_artifacts_path_commit/$universal_zip_file_name" >> "$GITHUB_OUTPUT"
+            echo universal_zip_url_latest="https://storage.googleapis.com/$gcs_artifacts_path_latest/$universal_zip_file_name" >> "$GITHUB_OUTPUT"
+            echo universal_zip_url_commit="https://storage.googleapis.com/$gcs_artifacts_path_commit/$universal_zip_file_name" >> "$GITHUB_OUTPUT"
```

Comment:
Unrelated fix for a bug discovered via act testing. I have updated it in this PR to make the tests pass. Bugfix has been split into its own PR: #459. I'll rebase once that PR is merged.